README.md

```diff
@@ -1,163 +1,74 @@
-  </a>
-  <a href="https://facebook.github.io/react-native/docs/contributing">
+  <a href="https://github.com/microsoft/react-native-macos/blob/master/CONTRIBUTING.md">
     <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
   </a>
-  <a href="https://twitter.com/intent/follow?screen_name=reactnative">
-    <img src="https://img.shields.io/twitter/follow/reactnative.svg?label=Follow%20@reactnative" alt="Follow @reactnative" />
-  </a>
 </p>
 
-<h3 align="center">
-  <a href="https://facebook.github.io/react-native/docs/getting-started">Getting Started</a>
-  <span> · </span>
-  <a href="https://facebook.github.io/react-native/docs/tutorial">Learn the Basics</a>
-  <span> · </span>
-  <a href="https://facebook.github.io/react-native/showcase.html">Showcase</a>
-  <span> · </span>
-  <a href="https://facebook.github.io/react-native/docs/contributing">Contribute</a>
-  <span> · </span>
-  <a href="https://facebook.github.io/react-native/en/help">Community</a>
-  <span> · </span>
-  <a href="https://github.com/facebook/react-native/blob/master/.github/SUPPORT">Support</a>
-</h3>
-
-React Native brings [**React**'s][r] declarative UI framework to iOS and Android. With React Native, you use native UI controls and have full access to the native platform.
-
-- **Declarative.** React makes it painless to create interactive UIs. Declarative views make your code more predictable and easier to debug.
-- **Component-Based.** Build encapsulated components that manage their own state, then compose them to make complex UIs.
-- **Developer Velocity.** See local changes in seconds. Changes to JavaScript code can be live reloaded without rebuilding the native app.
-- **Portability.** Reuse code across iOS, Android, and [other platforms][p].
-
-React Native is developed and supported by many companies and individual core contributors. Find out more in our [ecosystem overview][e].
-
-[r]: https://reactjs.org/
-[p]: https://facebook.github.io/react-native/docs/out-of-tree-platforms
-[e]: https://github.com/facebook/react-native/blob/master/ECOSYSTEM.md
+> See the official [React Native website](https://reactnative.dev/) for an introduction to React Native. 
+
+[React Native](https://reactnative.dev) is a framework developed by Facebook that enables you to build world-class application experiences on native platforms using a consistent developer experience based on JavaScript and [React](https://reactjs.org/). The focus of React Native is on developer efficiency across all the platforms you care about - learn once, write anywhere.
+
+This repository is a working fork of **facebook/react-native** that adds support for the official React Native for macOS implementation from Microsoft. 
+
+You can read more about the macOS implementation in our website - [React Native for Windows and Mac](https://microsoft.github.io/react-native-windows/)
 
 ## Contents
 
 - [Requirements](#-requirements)
```

Comment:
All of these intra-document links are broken except for 'Code of Conduct'.     I think simply removing the preceding '-', like `[Documention][#documentation]` will fix them

Reply:
Oops. Done.